ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java

```diff
@@ -146,6 +146,7 @@ public void getObjects() throws IOException, SignatureException, URISyntaxExcept
                     bucketName, "beowulf.txt"));
             assertThat(headResponse.getStatus(),
                     is(HeadObjectResponse.Status.EXISTS));
+            assertThat(headResponse.getObjectSize(), is(294059L));
```

Comment:
headObject should probably be its own test.  Doesn't really matter, since the functionality is covered either way.